packages/esm-patient-medications-app/src/add-drug-order/drug-search/order-basket-search-results.component.tsx

```diff
@@ -141,8 +141,10 @@ const DrugSearchResultItem: React.FC<DrugSearchResultItemProps> = ({ drug, openO
       // Directly adding the order to basket should be marked as incomplete
       searchResult.isOrderIncomplete = true;
       setOrders([...orders, searchResult]);
-      closeWorkspace('add-drug-order', true);
-      launchPatientWorkspace('order-basket');
+      closeWorkspace('add-drug-order', {
+        ignoreChanges: true,
```

Comment:
```suggestion
```

Per the above change

Reply:
This requires the `ignoreChanges: true`, since the order basket looks for unsaved orders, before it can be closed.
If there is an unsaved order and you try to open any other workspace (drug form/ lab form), it will show the prompt.

Reply:
The order basket looks for unsaved orders? I thought we're closing the add-drug-order workspace?

Reply:
My bad, actually I added the prompt for search as well, which you then suggested to remove.